py/issue_triage/triage.py

```diff
@@ -12,7 +12,15 @@
 import retrying
 import json
 
-TOKEN_NAME = "GITHUB_TOKEN"
+PROJECT_CARD_ID = os.getenv('INPUT_PROJECT_CARD_ID')
```

Comment:
Should we continue to supply MD2... as a default value e.g.

```
PROJECT_CARD_ID = os.getenv('INPUT_PROJECT_CARD_ID', MDEzOlByb2plY3RDb2x1bW41OTM0MzEz)
```

Reply:
This default value is passed from the workflow into the environment variable, so I think its better to keep things like this.   The default value is specified here: https://github.com/kubeflow/triage-issues/blob/master/action.yml#L8